astor/code_gen.py

```diff
@@ -459,12 +480,13 @@ def visit_Raise(self, node):
     def visit_Attribute(self, node):
         self.write(node.value, '.', node.attr)
 
-    def visit_Call(self, node):
+    def visit_Call(self, node, len=len):
+        write = self.write
```

Comment:
A very diminishing return micro-optimization.  visit_Call has a lot of calls to write, and is itself called a lot; the result of this is measurable, but it's not worth doing for any of the other visitors.